addons/repos/main.yaml

```diff
@@ -97,6 +97,7 @@ packages:
     versions:
       - 2.27.0
       - 2.36.2
+      - 2.37.0
```

Comment:
Since TCE is no longer updating and publishing a Package Repository, it is not necessary to update this file. Please remove this change.